integration/try/try.go

```diff
@@ -101,6 +102,36 @@ func (t *Try) WaitDeleteDeployment(name string, namespace string, timeout time.D
 	return nil
 }
 
+// WaitPodIPAssigned wait until the pod is assigned an IP.
+func (t *Try) WaitPodIPAssigned(name string, namespace string, timeout time.Duration) error {
+	ebo := backoff.NewExponentialBackOff()
+	ebo.MaxElapsedTime = applyCIMultiplier(timeout)
+
+	if err := backoff.Retry(safe.OperationWithRecover(func() error {
+		podList, err := t.client.ListPodWithOptions(namespace, metav1.ListOptions{})
```

Comment:
Since we are interested in the state of a single pod, could we simply add a `t.client.GetPodWithOptions` or directly call `t.client.KubeClient.CoreV1().Pods(namespace).Get(name, metav1.ListOptions{})` ?